Test-NetStack.psm1

```diff
@@ -603,7 +672,7 @@ Function Test-NetStack {
                             if ($thisSourceResult.ReceivedPctgOfLinkSpeed -ge $Definitions.NDKPerf.TPUT) { $Result | Add-Member -MemberType NoteProperty -Name PathStatus -Value 'Pass' }
                             else { $Result | Add-Member -MemberType NoteProperty -Name PathStatus -Value 'Fail' }
                         }
-                        else { throw 'Internal Error: Test-NetStack Data Integrity Issue in Stage4' }
+                        else { $Result | Add-Member -MemberType NoteProperty -Name PathStatus -Value 'Fail' }
```

Comment:
Same as above; the 'Fail' is correct, but the we should add the reason the the log file so we can distinguish this code path from a regular failure